cmake/CompilerWarnings.cmake

```diff
@@ -109,6 +109,21 @@ function(mamba_target_add_compile_warnings target)
         -Wuninitialized
     )
 
+    # It seems that these flags are leaked to CXXFLAGS: `-framework CoreFoundation` `-framework
+    # CoreServices` `-framework Security` `-framework Kerberos` `-fno-merge-constants`
```

Comment:
How is this comment related to the elements bellow? Can you precise where those are leaked? Is it when you build locally on macOS?

See also [those lines on the feedstock which make use of them](https://github.com/conda-forge/micromamba-feedstock/blob/326c9d4b3f78189f48b00771388a15ef1528e031/recipe/build.sh#L10-L14).

The use and presence of those flags must be clarified at a point, but this is orthogonal to the goal of this PR to me.